algorithms/bfs/Bfs.py

```diff
@@ -0,0 +1,48 @@
+from collections import defaultdict
+
+class Graph:
+ 
+    def __init__(self):
+        self.graph = defaultdict(list)
+ 
+    def addEdge(self,u,v):
+        self.graph[u].append(v)
+ 
+    def BFS(self, s):
```

Comment:
1. BFS -> bfs
2. we need more descriptive parameter. even better if you could use type hinting.
ex) 
```python
def bfs(self, s: string):
```